extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -137,11 +143,22 @@ export class AmpUserNotification extends AMP.BaseElement {
 
     this.storageKey_ = 'amp-user-notification:' + this.elementId_;
 
+    this.showIfGeo_ = this.element.getAttribute('data-show-if-geo');
+
     this.showIfHref_ = this.element.getAttribute('data-show-if-href');
     if (this.showIfHref_) {
       assertHttpsUrl(this.showIfHref_, this.element);
     }
 
+    user().assert(
+        !(this.showIfHref_ && this.showIfGeo_ !== null),
```

Comment:
nit: Fail to assign a value to `data-show-if-geo` will lead to the user-notification never show. Do we want to throw an user error and document it?